model/task_lifecycle.go

```diff
@@ -1251,23 +1256,28 @@ func removeNextMergeTaskDependency(cq commitqueue.CommitQueue, currentIssue stri
 }
 
 // evalStepback runs linear or bisect stepback depending on project, build variant, and task settings.
+// The status passed in is a display status, initially stepback only activates if the task
+// has failed but not on system failure.
 func evalStepback(ctx context.Context, t *task.Task, caller, status string, deactivatePrevious bool) error {
 	s, err := getStepback(t.Id)
 	if err != nil {
 		return errors.WithStack(err)
 	}
+	// A new stepback happens when the display status is failed (not system failure)
+	// and it is not aborted.
+	newStepback := status == evergreen.TaskFailed && !t.Aborted
```

Comment:
Since this is checking the display status, this looks like it would exclude other display statuses like `evergreen.TaskSetupFailed`/`evergreen.TaskTimedOut` from initiating stepback, but we only want to skip for abort + system failures. Can we have this just exclude aborted tasks and system-failed tasks specifically?

Reply:
Hm, I'm not entirely sure what statuses should exclude stepback or not- I simply kept the same activation conditions that were originally for linear stepback.

Looking in globals.go, we have a slice of `TaskSystemFailures` which includes TaskTimedOut- the only two we have that aren't apart of the TaskSystemFailures slice is TaskFailed & TaskSetupFailed.

Should I edit it to activate stepback for the 3 statuses (TaskFailed, TaskSetupFailed, and TaskTimedOut)- or just 2 (TaskFailed, TaskSetupFailed)?